shell/server/proxy.js

```diff
@@ -348,6 +353,7 @@ Meteor.methods({
 
     const token = params.token;
     const incognito = params.incognito;
+    const standalone = globalDb.getStandaloneDomainForToken(token);
```

Comment:
What's the purpose behind parentOrigin here? Do you mean for it to be passed to the Proxy and enforced there?